flow/connectors/eventhub/eventhub.go

```diff
@@ -53,11 +53,15 @@ func NewEventHubConnector(
 	}, nil
 }
 
-func (c *EventHubConnector) Close(ctx context.Context) error {
-	err := c.hubManager.Close(ctx)
-	if err != nil {
-		c.logger.Error("failed to close event hub manager", slog.Any("error", err))
-		return err
+func (c *EventHubConnector) Close() error {
+	if c != nil {
+		timeout, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
we have seen this take longer than 5 seconds from time to time, can we make this 30 seconds?

Reply:
Updated both to 1 minute